_posts/2017/2017-06-27-why-a-techwriter-should-write-your-api-docs.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "API writing - just copy and paste?"
+date: "2017-06-27 06:34:02"
+image: blog-header/api-journey.jpg
+categories: api
+authors: ["Birgit"]
+---
+
+Can you imagine that?
+There are people who think that API writing is just about copying and pasting text.
+Or that API writers just make things nice and shiny; polish some wording here and there.
+I'm an API writer and I was super sad to hear that.
+When I addressed this with some of my colleagues, they said, of course they know that this is **not** what I do.
+But for them API documentation was a very abstract topic and they were very interested in what exactly I do.
+So I took them on a little API docs journey for our brand new software...
+
```

Comment:
I like the cynical tone of the introduction and how you refer to the intro at the end of the post again 👍